cmd/proxy/Dockerfile

```diff
@@ -44,7 +45,8 @@ RUN chmod 644 /config/config.toml
 RUN apk add --update git git-lfs mercurial openssh-client subversion procps fossil tini
 
 # Add git-credential-github-app for native integration with GitHub Apps
-RUN wget -O git-credential-github-app.tar.gz https://github.com/bdellegrazie/git-credential-github-app/releases/download/v0.3.0/git-credential-github-app_v0.3.0_Linux_x86_64.tar.gz \
+RUN if [ "${TARGETARCH}" = "arm64" ]; then ARCH="arm64"; else ARCH="amd64"; fi \
```

Comment:
@nrwiersma I'm sorry I messed up this one, should've been x86_64 in the second case

Reply:
You can open a new PR, no stress. I should have caught this.